docs/stable/store/sllm-store-cli.md

```diff
@@ -0,0 +1,194 @@
+---
+sidebar_position: 2
+---
+
+# ServerlessLLM Store CLI
+
+ServerlessLLM Store's CLI allows the use `sllm-store`'s functionalities within a terminal window. It has the functions:
+- `save`: Convert a HuggingFace model into a loading-optimized format and save it to a local path.
+- `load`: Load a model into given GPUs.
+
+
+## Requirements
+- OS: Ubuntu 20.04
+- Python: 3.10
+- GPU: compute capability 7.0 or higher
+
+## Installations
+
+### Create a virtual environment
+```bash
+conda create -n sllm-store python=3.10 -y
+conda activate sllm-store
+```
+
+### Install C++ Runtime Library (required for compiling and running CUDA/C++ extensions)
+``` bash
+conda install -c conda-forge libstdcxx-ng=12 -y
+```
+
+### Install with pip
+```bash
+pip install serverless-llm-store
+```
+
+### Install from source
+1. Clone the repository and enter the `store` directory
+
+``` bash
+git clone https://github.com/ServerlessLLM/ServerlessLLM.git
+cd ServerlessLLM/sllm_store
+```
+
+2. Install the package from source
+
+```bash
+rm -rf build
+pip install .
+```
+
+## Getting Started
+### 1. Start the ServerlessLLM Store Server
+Firstly, start the ServerlessLLM Store server. By default, it uses ./models as the storage path.
+
+Launch the checkpoint store server in a separate process:
+``` bash
+# 'mem_pool_size' is the maximum size of the memory pool in GB. It should be larger than the model size.
+sllm-store start --storage-path $PWD/models --mem-pool-size 4GB
+```
+
+### 2. vLLM Patch (optional)
+
+To use vLLM with ServerlessLLM, you must apply a patch. This patch has been tested with vLLM version 0.9.0.1.
+
+1. **Check patch status** (optional):
+   ```bash
+   ./sllm_store/vllm_patch/check_patch.sh
+   ```
+
+2. **Apply the patch**:
+   ```bash
+   ./sllm_store/vllm_patch/patch.sh
+   ```
+
+3. **Remove the patch** (if needed):
+   ```bash
+   ./sllm_store/vllm_patch/remove_patch.sh
+   ```
+
+:::note
+The patch file is located at `sllm_store/vllm_patch/sllm_load.patch` in the ServerlessLLM repository.
+:::
+
+## Example Workflow
+1. Convert a model to ServerlessLLM format and save it to a local path:
+``` bash
+sllm-store save --model facebook/opt-1.3b --backend vllm
+```
+
+2. Load a previously saved model into memory, ready for inference:
```

Comment:
The example workflow is not complete. You can't load w/o starting the store server.